package.json

```diff
@@ -36,10 +37,12 @@
     "lodash.memoize": "^4.1.2",
     "loglevel-colored-level-prefix": "^1.0.0",
     "messageformat": "^1.0.2",
-    "prettier-eslint": "^8.5.0",
     "rxjs": "^5.3.0",
     "yargs": "10.0.3"
   },
+  "peerDependencies": {
+    "prettier-eslint": "^8.5.0"
```

Comment:
maybe loosen up a bit to "~8.5?"

Reply:
`^8.5.0` will match `8.5.0` or higher but lower than `9.0.0`
`~8.5` will match `8.5.x`

Reply:
@zimme doh! missed the ^.. getting tired over here :-)